.bit/responses/1.4-Week 1 Step 4.md

```diff
@@ -6,50 +6,49 @@ week: 1
 step: 4
 name: Week 1 Step 4
 ---
+Week 1 Step 4 ⬤⬤⬤⬤◯◯◯◯ | 🕐 Estimated completion: 35mins-45 mins
```

Comment:
The other issues don't have the filled in circles or estimated completion, so I would leave them out!